src/main/java/org/broadinstitute/consent/http/models/DatasetMetrics.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.consent.http.models;
+
+
+import java.util.List;
+
+import org.broadinstitute.consent.http.models.dto.DatasetDTO;
+
+public class DatasetMetrics {
+
+  private DatasetDTO dataset;
+  private List<DataAccessRequest> dars;
+  private List<Election> elections;
+
+  public DatasetDTO getDataset() {
+    return dataset;
+  }
```

Comment:
One way to lighten the response is to do more work here in the value class than in the display. For example, the UI doesn't really care about the full election or each complete DAR, it only needs some summary info and minimal calculations. Add each element individually, then when you are using the constructor, pass in a list of DARs and a list of Elections and populate the summary fields from those lists. Some suggestions:

* count of dars
* count of approved elections
* count of denied elections
* list of value objects, each containing fields for: 
  * DAR ID
  * Project Title
  * Last Submission (by the Researcher)
  * Principal Investigator
  * Non-Technical Summary

`Principal Investigator` is really one of the DAR's researcher properties, so I would suggest getting the DAR's researcher in addition to the DAR. The fact that it exists in the DAR's json is actually a problem that we need to not perpetuate.

Reply:
So, I'm just working on trying to get the PI. I'm looking at the user, that doesn't have PI on it, I'm looking at user properties, and that doesn't have the PI either that just has 'isPI' and 'hasPi' but it doesn't have a 'PI' property so for all the cases that the user is not the PI I don't know who it is. So, where can I find this information besides the dar?

Reply:
Look at the class `UserFields`. That has the names of all possible fields in user properties. Not all users have a PI, some of them **_ARE_** the PI. So, to get that value, we need to look at `isThePI`, and if false, then `piName` and `piEmail`. You can see all of these fields in action on your profile page.

Reply:
Okay thank you!

Reply:
@rushtong I'm wondering if either of these are a preferred approach (storing the elections vs storing the count of approved/denied)?  Something I am thinking about, although it is not included in the mockups, I wonder if we would want to know which dars have been approved/denied. We display the total number of dars, list of dar titles, the number approved, the number denied, and the dar non technical summaries. Would it be helpful to whoever is reading those summaries to know whether or not that dar was approved?

Reply:
Let's touch base with @solideoglori on this question: 
> Would it be helpful to whoever is reading those summaries to know whether or not that dar was approved?

Reply:
@solideoglori gave his thumb for this, said it would be helpful to know which were approved and denied.